src/fparser/two/tests/fortran2003/test_primary_r701.py

```diff
@@ -149,37 +149,38 @@ def test_array_constructor():
         expected_str='[1.2, 2.3E+2, - 5.1E-3]')
 
 
-@pytest.mark.xfail(reason="Matches Designator before Structure Constructor "
-                   "(issue #284)")
 @pytest.mark.usefixtures("f2003_create")
```

Comment:
There's a test in `test_fortran2003.py` that refers to R701.h. Perhaps move it to this file?

Reply:
OK, done.